uber/templates/registration/generate_promo_code.html

```diff
@@ -0,0 +1,50 @@
+{% extends "base-admin.html" %}
+{% block title %}Generate Promo Codes{% endblock %}
+{% block content %}
+
+<style>
+.decent > th,td{
+    padding: 1em;
+}
+.decent-row:hover{
+    background-color: #f5f5f5;
+}
+</style>
+
+<h2> Generate Promo Code </h2>
+
+<form method="post" action="generate_promo_codes">
+{% csrf_token %}
+<table class="form decent">
+<tr>
+    <td># of Codes: </td>
+    <td><input type="number" name="count" min="1" value="1" /></td>
```

Comment:
Setting static values is bad UX -- it means that if the form gets refreshed (like if we come back with a serverside validation error or if their connection gets interrupted), we undo everything the admin entered. If possible, we should pass the form data back to the form and use those for values. We can use the django `default` tag to fill in, well, default values.